### ISSUE TITLE
morph browser reboots my device

### ISSUE THREAD

**Issue by ksalver** [ubports/morph-browser#93]
when i open serveral tabs , sometime my device (BQ Aquaris 4.5) reboots immediately (ca. 5 -10 Minutes surfing) !!

**Comment by balcy** [comment]
ok then it is more important to solve https://github.com/ubports/morph-browser/issues/59 (currently an error prevents the tabs to be unloaded)

**Comment by balcy** [comment]
with https://github.com/ubports/morph-browser/pull/96 inactive tabs are unloaded so that should give more RAM back

**Comment by UniversalSuperBox** [comment]
It seems I can reproduce this *-ish-* by creating memory pressure on any device using `stress-ng`. The Linux Out Of Memory killer chooses to kill Unity8, which takes you back to the system spinner. The device does not actually reboot, but there isn't much difference between this and rebooting.

This isn't actually morph-browser's fault, but rather the system is just using too much memory. If Unity8 wasn't killed, the browser would be (and then people would say that the browser is crashing all the time).

**Comment by Stefano0101** [comment]
This is happening on Pro 5 while browsing some websites (not all the time), especially when opening a second page while selecting some search options on the first page. It opens new window, but instead of showing the selector field/s there is just a white page or there is a spinner for a while. When attempted to close this browser page, Morph crashes. Sometimes crashes when all browser tabs are closed as well. I tried to clear the cache, then it worked for one time and the back to the same crashing behavior. (Try rightmove.co.uk   looking for a property, it crashes always)
This is happening for a while now with Morph browser on Pro5. I'm on latest OTA-7, stable. Let me know if any logs are needed.

**Comment by balcy** [comment]
Yes I can confirm the problem with the selections. (sometimes the new page loads forever and closing it restarts unity8 / crashes the browser)
Hopefully solving https://github.com/ubports/morph-browser/issues/54 gets rid of that problem.

**Comment by lnoble237** [comment]
this happens a lot on FP2 especially while watching videos but also after viewing for a long time. can try to get crash logs
